openpype/pipeline/temporarydir.py

```diff
@@ -0,0 +1,65 @@
+"""
+Temporary folder operations
+"""
+
+import os
+from openpype.lib import StringTemplate
+from openpype.pipeline import Anatomy
+
+
+def create_custom_tempdir(project_name, anatomy=None, formating_data=None):
```

Comment:
There should be a function which create and return tempdir no matter if `OPENPYPE_TMPDIR` is set like in comment https://github.com/ynput/OpenPype/pull/4398#issuecomment-1422661943

Reply:
This is not resolved. The function does nothing if `OPENPYPE_TMPDIR` is not set. Also `project_name` should be required.

Reply:
You have to separate 2 things:
1. use different directory than default tmpdir by using `OPENPYPE_TMDIR` -> done
2. single function to create tempdir with prefix -> missing